gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitorFactory.java

```diff
@@ -49,24 +52,29 @@ public class KafkaJobStatusMonitorFactory implements Provider<KafkaJobStatusMoni
   private final JobIssueEventHandler jobIssueEventHandler;
   private final MultiContextIssueRepository issueRepository;
   private final boolean instrumentationEnabled;
+  private final DagActionStore dagActionStore;
+  private final boolean dagProcEngineEnabled;
```

Comment:
I didn't see this being used in `KafkaJobStatusMonitor`, so you don't seem to need to have it here.